Source/Core/VideoBackends/D3D/D3DState.cpp

```diff
@@ -301,7 +301,14 @@ ID3D11SamplerState* StateCache::Get(SamplerState state)
   sampdc.MinLOD = state.min_lod / 16.f;
   sampdc.MipLODBias = (s32)state.lod_bias / 32.0f;
 
-  if (state.anisotropic_filtering)
+  //if texture is marked as hacky
+  if(state.lod_bias_scale)
```

Comment:
There should be a space after the `if` otherwise the lint bot in our CI will flag you on this.